bittensor/_dataset/__init__.py

```diff
@@ -164,7 +194,7 @@ def add_defaults(cls, defaults):
         defaults.dataset.block_size = os.getenv('BT_DATASET_BLOCK_SIZE') if os.getenv('BT_DATASET_BLOCK_SIZE') != None else 20
```

Comment:
This should probably also get updated, 20 is the old seq_length on nobunaga. I'd actually prefer if we change this variable and leave it as `None` in the `new()` method, otherwise someone may change this line and accidentally not update it correctly.